modules/swagger-codegen/src/main/resources/JavaJaxRS/cxf-cdi/beanValidation.mustache

```diff
@@ -0,0 +1,53 @@
+{{#required}}
```

Comment:
OK - because this template is used for method annotations, you're adding newlines.  However, to my knowledge, there's no Java syntax requirement for newlines on method/property annotations - indeed, much like parameter annotations, newlines are about style, rather than syntax.

So - syntactically I think this would be OK without the newlines.  What do you think?

I've also noticed you've got some clever integer/decimal handling in here that's missing in `beanValidationParams.mustache`- should that be ported?